dev/workflow/build_opts.yaml

```diff
@@ -71,7 +71,7 @@ build:
   gsi_enkf:
     command: "./build_gsi_enkf.sh"
     cores: 8
-    walltime: "00:15:00"
+    walltime: "00:20:00"
```

Comment:
Looks, good.  I'll update this on disk and "push" the replay on last night's nightly.